python/desc/imsim/camera_readout.py

```diff
@@ -469,8 +469,12 @@ def write_fits_file(self, outfile, overwrite=True, run_number=None,
             Run number.  If None, then the visit number is used.
         compress: bool [True]
             Use RICE_1 compression for each image HDU.
-        image_type str ['SKYEXP']
+        image_type: str ['SKYEXP']
             Image type to write to the `OBSTYPE` and `IMGTYPE` keywords.
+        added_keywords: dict [None]
+            Python dict of key/value pairs to add to the primary HDU.
+            These will be set just before writing the FITS file, so
+            they will override any existing keywords.
```

Comment:
maybe add:

'they will override any existing keywords or be added to the header list if they don't exist.'

Reply:
The name of the option indicates that the keywords will be added and the first sentence of the description also says that they will be added.  I think it's probably ok as-is.